lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -58,12 +56,16 @@ def create_annotation_job(
     reference model should be used to generate annotations.
     See more: https://blog.mozilla.ai/lets-build-an-app-for-evaluating-llms/
     """
-    inference_job_create_request = JobInferenceCreate(
-        **job_create_request.dict(),
+    inference_job_create_config = JobAnnotateConfig(
+        **job_create_request.job_config.dict(),
         model="hf://facebook/bart-large-cnn",
         output_field="ground_truth",
     )
-    inference_job_create_request.store_to_dataset = True
+    inference_job_create_config.store_to_dataset = True
+    inference_job_create_request_dict = job_create_request.model_dump()
+    inference_job_create_request_dict.job_config = inference_job_create_config
+
+    inference_job_create_request = JobCreate(**inference_job_create_request_dict)
```

Comment:
```suggestion
    inference_job_create_request = JobCreate(
        **job_create_request.model_dump(), 
        job_config=inference_job_create_config
    )
```

Reply:
IIRC Pydantic complained because it found some keys twice (so no "key overwriting" was possible with this syntax). I'll check it out again.

Reply:
Ok, it did work O_o Updating and pushing.

Reply:
Yes, basically I get at https://github.com/mozilla-ai/lumigator/actions/runs/13139121520/job/36661667194?pr=740#step:7:932 :

```
>       inference_job_create_request = JobCreate(
            **job_create_request.model_dump(), job_config=inference_job_create_config_dict
        )
E       TypeError: lumigator_schemas.jobs.JobCreate() got multiple values for keyword argument 'job_config'
```

So I'll leave it as it is.